settings.gradle.kts

```diff
@@ -24,6 +24,8 @@ pluginManagement {
 include(
     "chart",
     "color",
+    "element",
     "kanvas",
-    "sample"
+    "sample",
+    "selection"
```

Comment:
Just FYI, if you upgrade the Gradle wrapper (not sure which version it was introduced, but 7.3 definitely has it) then you can use trailing commas in your Gradle `kts` files.